### PR TITLE
Review project

### PR DESCRIPTION
**Pull request by CesarHerr**
Review project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+          <li><a  class="nav-link" href="#What_you_should_already_know">What you should already know</a></li>
+          <li><a  class="nav-link" href="#JavaScript_and_Java">JavaScript and Java</a></li>
+          <li><a  class="nav-link" href="#Hello_world">Hello World</a></li>
+          <li><a  class="nav-link" href="#Variables">Variables</a></li>
+          <li><a  class="nav-link" href="#Declaring_variables">Declaring variables</a></li>
+          <li><a  class="nav-link" href="#Variable_scope">Variable Scope</a></li>
+        </ul>
+      </nav>
+
+    </nav>
```
Comment (CesarHerr) [inline index.html:27]: Great job working on this project 👍. Remember, The **header**  element represents introductory content, typically a group of introductory or navigational aids, kindly replace ** nav id="navbar" ** with a **header** tag in line with HTML semantic tags rules. 🥺

File: index.html [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    </p>
+    <p id="javaFunction">
+      <code>function greetMe(yourName) { alert("Hello " + yourName); }
+        greetMe("World");</code>
+    </p>
+    <p>
+      Select the code in the pad and hit Ctrl+R to watch it unfold in your browser!
+    </p>
+    </section>
+    <section class="main-section" id="Variables">  <header><h1>Variables</h1></header>
```
Comment (CesarHerr) [inline index.html:78]: Great job working on this project 👍. Remember, The **h1** element is for a top-level section, and you have to use just one on your page, use more than 1 is not considered a best practice.  [for more information...](https://adrianroselli.com/2016/08/there-is-no-document-outline-algorithm.html)    kindly replace for h2, h3, etc.. 🙏

File: index.html [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+    </p>
+    <p>
+      In contrast to Java's compile-time system of classes built by declarations, JavaScript supports a runtime system based on a small number of data types representing numeric, Boolean, and string values. JavaScript has a prototype-based object model instead of the more common class-based object model. The prototype-based model provides dynamic inheritance; that is, what is inherited can vary for individual objects. JavaScript also supports functions without any special declarative requirements. Functions can be properties of objects, executing as loosely typed methods.
+
+JavaScript is a very free-form language compared to Java. You do not have to declare all variables, classes, and methods. You do not have to be concerned with whether methods are public, private, or protected, and you do not have to implement interfaces. Variables, parameters, and function return types are not explicitly typed.
+    </p>
+    </section>
+    <section class="main-section" id="Hello_world">  <header><h1>Hello world</h1></header>
```
Comment (CesarHerr) [inline index.html:66]: Great job working on this project 👍. Remember, The h1 element is for a top-level section, and you have to use just one on your page, use more than 1 is not considered a best practice. [for more information...](https://adrianroselli.com/2016/08/there-is-no-document-outline-algorithm.html) kindly replace for h2, h3, etc.. 🙏

File: index.html [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+      JavaScript contains a standard library of objects, such as Array, Date, and Math, and a core set of language elements such as operators, control structures, and statements. Core JavaScript can be extended for a variety of purposes by supplementing it with additional objects; for example:
+    </p>
+    <p>
+      <ul>
+      <li>Client-side JavaScript extends the core language by supplying objects to control a browser and its Document Object Model (DOM). For example, client-side extensions allow an application to place elements on an HTML form and respond to user events such as mouse clicks, form input, and page navigation.</li>
+      <li>Server-side JavaScript extends the core language by supplying objects relevant to running JavaScript on a server. For example, server-side extensions allow an application to communicate with a database, provide continuity of information from one invocation to another of the application, or perform file manipulations on a server.</li>
+     </ul>    
+   </p>
+    </section>
+    <section class="main-section" id="What_you_should_already_know"> <header><h1>What you should already know</h1></header>
```
Comment (CesarHerr) [inline index.html:44]: Great job working on this project 👍. Remember, The h1 element is for a top-level section, and you have to use just one on your page, use more than 1 is not considered a best practice. [for more information...](https://adrianroselli.com/2016/08/there-is-no-document-outline-algorithm.html) kindly replace for h2, h3, etc.. 🙏

File: index.html [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+    </p>
+    <p>
+      <ul>
+      <li>A general understanding of the Internet and the World Wide Web (WWW).</li>
+      <li> Good working knowledge of HyperText Markup Language (HTML).</li>
+      <li>Some programming experience. If you are new to programming, try one of the tutorials linked on the main page about JavaScript.</li>
+    </ul>
+    </p>
+    </section>
+    <section class="main-section" id="JavaScript_and_Java"> <header><h1>JavaScript and Java</h1></header>
```
Comment (CesarHerr) [inline index.html:56]: Great job working on this project 👍. Remember, The h1 element is for a top-level section, and you have to use just one on your page, use more than 1 is not considered a best practice. [for more information...](https://adrianroselli.com/2016/08/there-is-no-document-outline-algorithm.html) kindly replace for h2, h3, etc.. 🙏

File: index.html [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+    <p>
+      You use variables as symbolic names for values in your application. The names of variables, called identifiers, conform to certain rules.
+
+A JavaScript identifier must start with a letter, underscore (_), or dollar sign ($); subsequent characters can also be digits (0-9). Because JavaScript is case sensitive, letters include the characters "A" through "Z" (uppercase) and the characters "a" through "z" (lowercase).  
+    </p>
+    <p>
+      You can use ISO 8859-1 or Unicode letters such as å and ü in identifiers. You can also use the Unicode escape sequences as characters in identifiers. Some examples of legal names are Number_hits, temp99, and _name.
+    </p>
+    </section>
+    <section class="main-section" id="Declaring_variables">  <header><h1>Declaring variables</h1></header>
```
Comment (CesarHerr) [inline index.html:88]: Great job working on this project 👍. Remember, The h1 element is for a top-level section, and you have to use just one on your page, use more than 1 is not considered a best practice. [for more information...](https://adrianroselli.com/2016/08/there-is-no-document-outline-algorithm.html) kindly replace for h2, h3, etc.. 🙏

File: index.html [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+    <code>
+      let y = 13.
+    </code><br><br>
+    <p>
+      This syntax can be used to declare a block scope local variable. See Variable scope below.
+    </p>
+  
+    </section>
+    
+    <section class="main-section" id="Variable_scope">  <header><h1>Variable scope</h1></header>
```
Comment (CesarHerr) [inline index.html:118]: Great job working on this project 👍. Remember, The h1 element is for a top-level section, and you have to use just one on your page, use more than 1 is not considered a best practice. [for more information...](https://adrianroselli.com/2016/08/there-is-no-document-outline-algorithm.html) kindly replace for h2, h3, etc.. 🙏

File: index.html [hunk 8]
```diff
@@ -0,0 +1,143 @@
...
+    </code><br>
+    </section>
+  
+      
+     </div>div   
+    
+  </main>
+    <script  src="js/index.js"></script>
+  </body>
+</html>
```
Comment (CesarHerr) [inline index.html:143]: Great job working on this project 👍. Remember that it is important to respect the indentation in the code. It is the best way to follow the best practices, make your code looks much better and cleaner, and easily readable. 😃

File: styles.css [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+    }
+          
+    
+    ul, code {
+        margin-left: 0;
+    }
+    
+  
+
+}
```
Comment (CesarHerr) [inline styles.css:53]: Great job working on this project 👍. Remember that it is important to respect the indentation in the code. It is the best way to follow the best practices, make your code looks much better and cleaner, and easily readable. 😃

**Review by CesarHerr — COMMENTED** [review]
Hi @CesarHerr ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Required Changes ♻️

- Kindly remember Linter should be set up correctly for this repository, make sure that there are no errors, linters results should be green.
- Kindly remember to use descriptive commit messages, the most important rule in maintaining a professional repo is to always think about the people that you share your code with.
-  Kindly remember to use a descriptive pull request title and summary, is important to give detailed information to your code reviewer.
- kindly remember to add a descriptive README file to your project.

## Optional suggestions

*Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.*

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

***Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.***

---

*As described in the [[Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [[this form](https://airtable.com/shrQAqnBwek5a0O0s)](https://airtable.com/shrQAqnBwek5a0O0s).*

**Review by shella12 — APPROVED** [review]
Hi @CesarHerr , 


## Status : APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Excellent review ✔️ 
- Great summary 👍 
- Good use of github review feature 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**